### ISSUE TITLE
Previewing/saving content with ${ produces a "403 Forbidden" error

### ISSUE THREAD

**Issue by spiralofhope** [skrysmanski/blogtext#37]
- PHP 7.3.23
- WordPress 5.6
- BlogText 0.9.9

### Steps to reproduce

- Begin a new post
- Type `${`
- Click "preview" on the right-hand side

A new tab opens, with the message:
```

403
Forbidden

Access to this resource on the server is denied!
```

### Notes

- This precise message is likely produced by my hosting and could be different for others.
- This is particularly damaging to my installation as I have a lot of inline code, and my shell scripting uses a `${variable}` construct.
- This problem did not exist on previous versions of BlogText.
  -  I know it did not exist on 2018-03-07 - 0.9.7 on WordPress 4.9.4 on PHP 5.6.33
  -  I'm pretty sure it did not exist on 2018-04-05 - 0.9.7 on WordPress 4.9.5 on PHP 5.6.34
  -  I do not know if it was introduced in 0.9.8 or the most recent 0.9.9.  I have upgraded through various versions of WordPress and PHP and have not kept close track of that and this bug.
- I don't know if this is related to #33 - `Percent character previews 403 Forbidden`

**Comment by spiralofhope** [comment]
I can no longer reproduce this in:

- BlogText  0.9.9
- PHP  7.3.27
- WordPress  5.7

**Comment by spiralofhope** [comment]
I'm just going to note that I reproduced this with a combination of `${` as well as having the text `.bashrc` within the page.

It's this alongside other content, but I'm unable to narrow it down.  This is insanity.

I solved this by carving a block of code out into another page, and replacing `.bashrc` and it's period (`.`) with the unicode [U+2024](https://www.compart.com/en/unicode/U+2024) (`․`) to make `․bashrc`.  Somehow that became valid.

I have no idea what's happening.  This is just a note for my future self.
